assets/dev/js/frontend/utils/assets-loader.js

```diff
@@ -29,11 +29,9 @@ export default class AssetsLoader {
 	}
 
 	isAssetLoaded( assetData, assetType ) {
-		const tag = 'script' === assetType ? 'script' : 'link',
-			filePath = `${ tag }[src="${ assetData.src }"]`,
```

Comment:
There was an error in this code. It checked for `link[src=...]` instead of `link[href=...]`.